stdlib/public/core/GroupInfo.json

```diff
@@ -49,6 +49,7 @@
     "Indices.swift",
     "Existential.swift",
     "WriteBackMutableSlice.swift",
+    "UnfoldSequence.swift",
```

Comment:
I have no clue at all what GroupInfo.json is. I stuck this in here because the compiler complained otherwise. I hope I put it in the right place.